packages/document/docs/zh/api/config/config-build.mdx

```diff
@@ -206,34 +206,26 @@ export default defineConfig({
 
 ### markdown.highlightLanguages
 
-- Type: `(string | [string, string])[]`
+- Type: `[string, string][]`
 - Default:
 
 ```js
 const DEFAULT_HIGHLIGHT_LANGUAGES = [
   ['js', 'javascript'],
   ['ts', 'typescript'],
   ['jsx', 'tsx'],
-  'tsx',
-  'json',
-  'css',
-  'scss',
-  'less',
   ['xml', 'xml-doc'],
-  'diff',
-  'yaml',
   ['md', 'markdown'],
   ['mdx', 'tsx'],
-  'bash',
```

Comment:
This is an option-in feature(automatically detecting users highlight code block), right? 
It seems that it affects users currently using rspress and is it a breaking change?

Reply:
No, currently highlightLanguages only serve as aliases and only extract values of array types, while strings are not affected. This is imperceptible to users.